test/utils.py

```diff
@@ -345,6 +345,18 @@ def revert_liger_kernel_to_phi3(model_config: MiniModelConfig):
     print("Liger kernel patches have been reverted.")
 
 
+def revert_liger_kernel_to_olmo2(model_config: MiniModelConfig):
+    """
+    Revert all Liger kernel patches applied to Olmo2.
+    """
+
+    from transformers.models.olmo2 import modeling_olmo2
+
+    importlib.reload(modeling_olmo2)
+    model_config.model_class = modeling_olmo2.Olmo2ForCausalLM
```

Comment:
Wondering if we're monkey patching nn, shouldn't we also be reloading nn for all models 🤔

Reply:
I think we should, I'll probably work on it in a separate PR

Reply:
yeahh good catch

Reply:
Related: https://github.com/linkedin/Liger-Kernel/issues/315